src/pages/notes/david-copperfield/index.mdx

```diff
@@ -19,22 +19,108 @@ import HowToCite from "/src/components/HowToCite.astro";
+_David Copperfield_ was published serially in twenty monthly parts by Bradbury & Evans from May 1849 to November 1850. Following _Dombey and Son_ (1846-48), it is the second novel for which Dickens kept a complete set of Working Notes. These Notes, which are bound with the manuscript of the novel, were originally bequeathed to Dickens’s friend and later biographer John Forster, and are now kept in the National Art Library at the Victoria and Albert Museum in London. The only exception is the Working Note for the first installment, which was removed from the manuscript for use in Forster’s _Life of Charles Dickens_ (1872-74); this sheet is bound with the Forster manuscript in the same library. 
 
 <ImageWithCaption
   src="https://upload.wikimedia.org/wikipedia/commons/5/50/Copperfield_cover_serial.jpg"
   alt="David Copperfield first serial installment cover"
-  caption="David Copperfield No. I (May 1849)"
+  caption="David Copperfield No. I (May 1849). Although Dickens considered several different titles, his eighth novel was ultimately published under the full title: The Personal History, Adventures, Experience, and Observation of David Copperfield the Younger of Blunderstone Rookery (Which He never meant to be Published on any Account)."
+/>
+
+The DDNP's annotations to the _David Copperfield_ Working Notes explore Dickens’s varied use of the Working Note system throughout the nineteen-month period of the novel’s composition. In particular, we attend to Dickens’s tendency to use the Working Notes not as plans, but as retroactive records of the events and developments of installments that had already been written. Although this practice allowed for a more open and spontaneous approach throughout most of the novel's composition, the last four Working Notes indicate Dickens’s adoption of a more rigorous and methodical approach to the final quarter of the novel in order to tie up its several narrative threads. 
+
+## Titling the Novel: _Copperfield_'s Genre and Form
+
+As he began “revolving a new work” in the early months of 1849 (Letters 5.489), Dickens wrote to Forster that he was experiencing the “deepest despondency [...] in commencing” and selecting a name for his new novel (Forster 2.77). As he had done with _Martin Chuzzlewit_ (1843-44), and as he did again with _Bleak House_ (1852-53), Dickens documented possible titles on several pages of notepaper that were later bound, along with the Working Notes, into the novel’s manuscript. The sixteen title sheets that accompany the _Copperfield_ manuscript provide insight into a difficult titling process, and into Dickens’s changing attitude towards his eighth novel’s genre and narrative method in the early stages of composition. 
+
+<ImageWithCaption
+  src="/images/david_copperfield/DC_titlepage.jpg"
+  alt="Sample title page for David Copperfield"
+  caption="One of sixteen sheets where Dickens tested titles for David Copperfield, © Victoria & Albert Museum, London"
```

Comment:
I think we should add a URL here to the V&A source for this image:  https://collections.vam.ac.uk/item/O1491532/personal-history-and-experience-of-manuscript-charles-dickens/